tsconfig.base.json

```diff
@@ -17,8 +17,7 @@
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": false,
     "skipLibCheck": true,
-    "lib": ["es2015"],
-    "types": ["jest"]
+    "lib": ["es2015"]
```

Comment:
Why did you remove `jest` from the types?

Reply:
`npm run clean-build` was complaining about unknown types. When you define the `types` array, all other `@types/` dependencies are not implicitly loaded like they are when `types` is not defined.

Reply:
When building it should use the `tsconfig.build.json`. I guess the update of typescript brought this error. With previous version, extending a tsconfig would not bring the `types` array.

Reply:
Actually if you update typescript version from `2.x` to `3.x`, make it clear in your PR title.

Reply:
I can update the title if you wish, but I upgraded to `3.x` after I opened the PR and CI failed. Surprisingly, removing `"types"` was the only change required for `npm run clean-build` to work with `3.x`.